PIL/Image.py

```diff
@@ -632,6 +677,7 @@ def _repr_png_(self):
 
     @property
     def __array_interface__(self):
+        # type: () -> Dict[Text, Any]
```

Comment:
According to the [NumPy documentation on `__array_interface__`](https://docs.scipy.org/doc/numpy/reference/arrays.interface.html), and looking over the source file (particularly at `_conv_type_shape`), I think this particular `Any` could be constrained further to `Tuple[Tuple[int, ...], Text, int, bytes]`.